styleguide/templatetags/styleguide_tags.py

```diff
@@ -48,10 +52,10 @@ def do_example(parser, token):
     # Reset to the original parser which hasn't been consumed from yet
     parser = parser_copy
 
-    # Consume from this parser to create the example node we can render
-    tag_name, *args = token.split_contents()
-    args = list(args)
-    kwargs = {}
+    # # Consume from this parser to create the example node we can render
```

Comment:
Can you move this comment up and then remove the commented-out lines?

Reply:
The comment actually doesn't apply anymore, so I'm just removing all the commented lines.